lib/watch/index.js

```diff
@@ -4,34 +4,32 @@ const path = require('path')
 const cp = require('child_process')
 const chalk = require('chalk')
 const { arrayToRegExp } = require('./utils')
-
```

Comment:
the changes in this file are necessary because otherwise `watch` uses globals which prevent multiple tests executing watch from running. I moved everything to local within the watch function